apis/flinkcluster/v1beta1/flinkcluster_types.go

```diff
@@ -648,6 +648,9 @@ type FlinkClusterComponentsStatus struct {
 	// The state of configMap.
 	ConfigMap FlinkClusterComponentState `json:"configMap"`
 
+	// The state of PodDisruptionBudget.
+	PodDisruptionBudget FlinkClusterComponentState `json:"podDisruptionBudget"`
```

Comment:
@kassybas hm, maybe we don't need to expose this at the moment; we can just go ahead and have a default disruption budget of 0%. This can later change when we look into reactive mode et al.